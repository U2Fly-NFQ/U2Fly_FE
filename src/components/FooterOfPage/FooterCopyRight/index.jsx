import React from 'react'
import './index.scss'
export default function FooterCopyRight() {
  return (
    <div className="footer-copyright wide grid">
      <div className="footer-copyright">
        <p>Copyright Â© 2022 U2Fly Booking Designed by U2FlyTeam</p>
      </div>
    </div>
  )
}
