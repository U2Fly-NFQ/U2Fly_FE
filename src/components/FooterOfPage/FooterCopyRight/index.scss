.footer-copyright {
  margin-top: 20px;

  .footer-copyright {
    text-align: center;
  }
}
