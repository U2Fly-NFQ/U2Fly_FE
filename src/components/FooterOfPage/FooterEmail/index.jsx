import { Col, Row } from 'antd'
import React from 'react'
import emailImage from '../../../assets/images/system/email.png'
import ButtonOfPage from '../../ButtonOfPage'
import InputOFPage from '../../InputOfPage'
import './index.scss'
export default function FooterEmail() {
  return (
    <div className="footer-email">
      <Row style={{ alignItems: 'center' }}>
        <Col span={12}>
          <div className="footer-email__logo-content">
            <div className="footer-email__logo-content__logo">
              <img src={emailImage} />
            </div>
            <div className="footer-email__logo-content__content">
              <h4>Get the latest news and offers</h4>
              <h2>Subscribe to our newsletter</h2>
            </div>
          </div>
        </Col>
        <Col span={12}>
          <div className="footer-email__email-send">
            <InputOFPage placeholder="Enter your email address" />
            <ButtonOfPage title="Subcribe" />
          </div>
        </Col>
      </Row>
    </div>
  )
}
