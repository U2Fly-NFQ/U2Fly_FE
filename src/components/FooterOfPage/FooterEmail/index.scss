.footer-email {
  padding-top: 10px;
  padding-bottom: 30px;
  background-color: #f3f6fd;
  margin-top: 100px;
  &__logo-content {
    &__logo {
      width: fit-content;
      img {
        width: 70%;
      }
    }
    &__content {
      font-size: 18px;
    }
    display: flex;
    align-items: center;
    // justify-content: center;
  }
  &__email-send {
    display: flex;
    align-items: center;
    .form-submit {
      padding: 0;
      button {
        border-radius: 0 8px 8px 0;
      }
    }
  }
}
