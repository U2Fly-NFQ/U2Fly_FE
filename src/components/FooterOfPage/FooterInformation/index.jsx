import { Col, Row } from 'antd'
import React from 'react'
import './index.scss'
import { useTranslation } from 'react-i18next'

export default function FooterInformation() {
  const { t } = useTranslation()

  return (
    <div className="footer-info  wide grid">
      <div className="footer-info__container ">
        <Row justify="space-between">
          <Col
            lg={{ span: 6 }}
            md={{ span: 8 }}
            sm={{ span: 24 }}
            xs={{ span: 24 }}
          >
            <div className="footer-info__container__contact">
              <div className="footer-info__container__title">
                <h5>{t('footer.Need any help?')}</h5>
              </div>
              <div className="footer-info__container__contact-content">
                <div className="footer-info__container__contact-content__item">
                  <h5>{t('footer.Call 24/7 for any help')}</h5>
                  <h3>
                    <a href="#">+00 123 456 789</a>
                  </h3>
                </div>
                <div className="footer-info__container__contact-content__item">
                  <h5>{t('footer.CMail to our support team')}</h5>
                  <h3>
                    <a href="#">support@domain.com</a>
                  </h3>
                </div>
                <div className="footer-info__container__contact-content__item">
                  <h5>{t('footer.Call 24/7 for any help')}</h5>
                  <h3>
                    <a href="#">+00 123 456 789</a>
                  </h3>
                </div>
              </div>
            </div>
          </Col>
          <Col
            lg={{ span: 3 }}
            md={{ span: 6 }}
            sm={{ span: 12 }}
            xs={{ span: 12 }}
          >
            <div className="footer-info__container__company">
              <div className="footer-info__container__title">
                <h5>{t('footer.Company')}</h5>
              </div>
              <div className="footer-info__container__company__content">
                <ul>
                  <li>{t('footer.About Us')}</li>
                  <li>{t('footer.Testimonials')}</li>
                  <li>{t('footer.Rewards')}</li>
                  <li>{t('footer.Work with Us')}</li>
                  <li>{t('footer.Meet the Team')}</li>
                  <li>{t('footer.Blog')}</li>
                </ul>
              </div>
            </div>
          </Col>
          <Col
            lg={{ span: 4 }}
            md={{ span: 6 }}
            sm={{ span: 12 }}
            xs={{ span: 12 }}
          >
            <div className="footer-info__container__company">
              <div className="footer-info__container__title">
                <h5>{t('footer.Support')}</h5>
              </div>
              <div className="footer-info__container__company__content">
                <ul>
                  <li>{t('footer.Account')}</li>
                  <li>{t('footer.Faq')} </li>
                  <li>{t('footer.Legal')}</li>
                  <li>{t('footer.Contact')}</li>
                  <li>{t('footer.Affiliate Program')}</li>
                  <li>{t('footer.Privacy Policy')}</li>
                </ul>
              </div>
            </div>
          </Col>
          <Col
            lg={{ span: 4 }}
            md={{ span: 5 }}
            sm={{ span: 12 }}
            xs={{ span: 12 }}
          >
            <div className="footer-info__container__company">
              <div className="footer-info__container__title">
                <h5>{t('footer.Other Services')}</h5>
              </div>
              <div className="footer-info__container__company__content">
                <ul>
                  <li>{t('footer.Community program')}</li>
                  <li>{t('footer.Investor Relations')}</li>
                  <li>{t('footer.Rewards Program')}</li>
                  <li>{t('footer.Partners')}</li>
                  <li>{t('footer.PointsPLUS')}</li>
                  <li>{t('footer.Blog')}</li>
                </ul>
              </div>
            </div>
          </Col>
          <Col
            lg={{ span: 5 }}
            md={{ span: 6 }}
            sm={{ span: 12 }}
            xs={{ span: 12 }}
          >
            <div className="footer-info__container__company">
              <div className="footer-info__container__title">
                <h5>{t('footer.Top cities')}</h5>
              </div>
              <div className="footer-info__container__company__content">
                <ul>
                  <li>Chicago</li>
                  <li>New York</li>
                  <li>San Francisco</li>
                  <li>California</li>
                  <li>Ohio</li>
                  <li>Alaska</li>
                </ul>
              </div>
            </div>
          </Col>
        </Row>
      </div>
    </div>
  )
}
