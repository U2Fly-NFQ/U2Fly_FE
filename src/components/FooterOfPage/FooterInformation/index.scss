@use '../../../assets/scss/abstracts/' as b;
.footer-info {
}
.footer-info__container {
  &__contact {
    &-content__item {
      padding-top: 5px;
      border-left: 2px solid b.$main-color;
      margin-top: 25px;
      padding-left: 9px;
      h5 {
        font-size: 16px;
        font-weight: 500;
      }
      h3 {
        font-weight: 500;
        font-size: 20px;
        padding-top: 0px;

        a {
          color: b.$main-color;
          text-decoration: underline;
        }
      }
    }
  }
  &__title {
    border-bottom: 1px solid b.$main-color;
    width: fit-content;
    font-size: 22px;
  }
  &__company {
    &__content {
      padding-top: 20px;
      ul {
        li {
          font-size: 14px;
          font-weight: 500;

          padding-bottom: 20px;
        }
      }
    }
  }
}
