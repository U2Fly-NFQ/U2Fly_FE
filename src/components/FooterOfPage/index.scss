.contact-info {
  background-color: #fff;
}
.copyright-info {
  background-color: #f3f6fd;
}
