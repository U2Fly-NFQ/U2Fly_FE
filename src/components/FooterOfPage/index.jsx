import React from 'react'
import FooterCopyRight from './FooterCopyRight'
import FooterInformation from './FooterInformation'

export default function FooterOfPage() {
  return (
    <>
      <div className="contact-info ">
        <div className="wide grid">
          <FooterInformation />
          <FooterCopyRight />
        </div>
      </div>
    </>
  )
}
