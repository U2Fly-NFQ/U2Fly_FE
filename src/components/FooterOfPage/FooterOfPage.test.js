import { I18nextProvider } from 'react-i18next'
import i18n from '../../translations'
import renderer from 'react-test-renderer'
import React, { Suspense } from 'react'
import { BrowserRouter as Router } from 'react-router-dom'

import FooterOfPage from './'

describe('Booking Passenger bar test', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <I18nextProvider i18n={i18n}>
          <Router>
            <Suspense>
              <FooterOfPage />
            </Suspense>
          </Router>
        </I18nextProvider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
