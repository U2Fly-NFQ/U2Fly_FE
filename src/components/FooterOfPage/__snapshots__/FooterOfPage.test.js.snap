// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Booking Passenger bar test renders correctly 1`] = `
<div
  className="contact-info "
>
  <div
    className="wide grid"
  >
    <div
      className="footer-info  wide grid"
    >
      <div
        className="footer-info__container "
      >
        <div
          className="ant-row ant-row-space-between"
          style={Object {}}
        >
          <div
            className="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-8 ant-col-lg-6"
            style={Object {}}
          >
            <div
              className="footer-info__container__contact"
            >
              <div
                className="footer-info__container__title"
              >
                <h5>
                  Need any help?
                </h5>
              </div>
              <div
                className="footer-info__container__contact-content"
              >
                <div
                  className="footer-info__container__contact-content__item"
                >
                  <h5>
                    CMail to our support team
                  </h5>
                  <h3>
                    u2fly@nfq.asia
                  </h3>
                </div>
                <div
                  className="footer-info__container__contact-content__item"
                >
                  <h5>
                    Call 24/7 for any help
                  </h5>
                  <h3>
                    028 6681 2733
                  </h3>
                </div>
              </div>
            </div>
          </div>
          <div
            className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-6 ant-col-lg-3"
            style={Object {}}
          >
            <div
              className="footer-info__container__company"
            >
              <div
                className="footer-info__container__title"
              >
                <h5>
                  Company
                </h5>
              </div>
              <div
                className="footer-info__container__company__content"
              >
                <ul>
                  <li>
                    About Us
                  </li>
                  <li>
                    Testimonials
                  </li>
                  <li>
                    Rewards
                  </li>
                  <li>
                    Work with Us
                  </li>
                  <li>
                    Meet the Team
                  </li>
                  <li>
                    Blog
                  </li>
                </ul>
              </div>
            </div>
          </div>
          <div
            className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-6 ant-col-lg-4"
            style={Object {}}
          >
            <div
              className="footer-info__container__company"
            >
              <div
                className="footer-info__container__title"
              >
                <h5>
                  Support
                </h5>
              </div>
              <div
                className="footer-info__container__company__content"
              >
                <ul>
                  <li>
                    Account
                  </li>
                  <li>
                    Faq
                     
                  </li>
                  <li>
                    Legal
                  </li>
                  <li>
                    Contact
                  </li>
                  <li>
                    Affiliate Program
                  </li>
                  <li>
                    Privacy Policy
                  </li>
                </ul>
              </div>
            </div>
          </div>
          <div
            className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-5 ant-col-lg-4"
            style={Object {}}
          >
            <div
              className="footer-info__container__company"
            >
              <div
                className="footer-info__container__title"
              >
                <h5>
                  Other Services
                </h5>
              </div>
              <div
                className="footer-info__container__company__content"
              >
                <ul>
                  <li>
                    Community program
                  </li>
                  <li>
                    Investor Relations
                  </li>
                  <li>
                    Rewards Program
                  </li>
                  <li>
                    Partners
                  </li>
                  <li>
                    PointsPLUS
                  </li>
                  <li>
                    Blog
                  </li>
                </ul>
              </div>
            </div>
          </div>
          <div
            className="ant-col ant-col-xs-12 ant-col-sm-12 ant-col-md-6 ant-col-lg-5"
            style={Object {}}
          >
            <div
              className="footer-info__container__company"
            >
              <div
                className="footer-info__container__title"
              >
                <h5>
                  Top cities
                </h5>
              </div>
              <div
                className="footer-info__container__company__content"
              >
                <ul>
                  <li>
                    Chicago
                  </li>
                  <li>
                    New York
                  </li>
                  <li>
                    San Francisco
                  </li>
                  <li>
                    California
                  </li>
                  <li>
                    Ohio
                  </li>
                  <li>
                    Alaska
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="footer-copyright wide grid"
    >
      <div
        className="footer-copyright"
      >
        <p>
          Copyright Â© 2022 U2Fly Booking Designed by U2FlyTeam
        </p>
      </div>
    </div>
  </div>
</div>
`;
